marketplaceItems/appreciate/appResources/appData/resources/js/appreciate_ui.js

```diff
@@ -0,0 +1,113 @@
+/*
+    Appreciate
+    Created by Zach Fox on 2019-01-30
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+/* globals document EventBridge setTimeout */
+
```

Comment:
Noticed the functions don't have our usual comments.   In terms of coding standards, do you think we should treat the HTML JS files different from the interface js files?

Reply:
oof, good point, no, i'll add comments